src/components/kmlLayer.js

```diff
@@ -32,10 +32,6 @@ export default mapElementFactory({
   name: 'kmlLayer',
   ctr: () => google.maps.KmlLayer,
 
-  inject: {
-
-  },
-
   render (h) {
```

Comment:
@hornetshell sorry, I surelly wasn't clear. I refered to all empty methods
```js
    // no-op
  },

  destroyed () {
    // no-op
  },

  beforeCreate (options) {
    // no-op
  },

  afterCreate (inst) {
    // no-op
  }
```
just left what the component needs.